### PR TITLE
Fix TembaSerializationException stringification

### LINKED ISSUE #65: TembaSerializationException always has the message <exception str() failed>
Hello,

I was using the `TembaObject.deserialize()` method to deserialize archived `Run` objects in my project (https://github.com/JustFixNYC/tenants2/pull/902) and noticed that an unusual exception was occurring.  It's easily reproducible in the REPL:

```
>>> from temba_client.serialization import TembaSerializationException
>>>
>>> raise TembaSerializationException('boop')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
temba_client.exceptions.TembaSerializationException: <exception str() failed>
```

The actual cause of `str()` failing can be found via:

```
>>> str(TembaSerializationException('boop'))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/venv/lib/python3.8/site-packages/temba_client/exceptions.py", line 3, in __str__
    return self.message
AttributeError: 'TembaSerializationException' object has no attribute 'message'
```

So it looks like `.message` isn't being set in the constructor for `TembaSerializationException`?  For now I'm just catching the exceptions and logging their `args[0]` attribute, but fixing these so they have meaningful error messages would be helpful.

### PR DESCRIPTION
**Pull request by toolness**
This should fix #65!

One funky thing is that running `nosetests --with-coverage --cover-erase --cover-package=temba_client --cover-html` didn't seem to actually find any tests, nor did running just `nosetests` on its own.  I ended up manually running test suites via e.g. `nosetests temba_client/tests.py` and that worked.

### CODE REVIEW

File: temba_client/exceptions.py [hunk 1]
```diff
@@ -56,7 +56,8 @@ def __str__(self):
 
 
 class TembaSerializationException(TembaException):
-    pass
+    def __init__(self, message):
+        self.message = message
```
Comment (toolness) [inline temba_client/exceptions.py:60]: I wanted to call `super()` but I wasn't sure what Python version y'all supported, and then I noticed that other `TembaException` subclasses that overrode `__init__()` never called `super()`, so I avoided it too... 🤷‍♂

Comment (rowanseymour) [reply temba_client/exceptions.py:60]: We dropped support for Python 2 a while back but never made the code more Python 3-ish

**Comment by rowanseymour** [comment]
Thanks!

**Comment by rowanseymour** [comment]
I can't replicate your problem with the tests.. that works for me with nosetests 1.3.7 and Python 3.7.4

**Pull request merged**